.github/workflows/tests.yml

```diff
@@ -0,0 +1,58 @@
+---
+name: Tests
+
+on:
+  push:
+    branches-ignore:
+      - 'master'
```

Comment:
Tests are required on push to `master` also.

Reply:
Fixed.